src/Apps/Artist/Routes/WorksForSale/Components/ZeroState.tsx

```diff
@@ -14,18 +13,9 @@ export const ZeroState: React.FC<React.PropsWithChildren<BoxProps>> = props => {
 
   return (
     <Box width="100%" {...props}>
-      <Sticky>
```

Comment:
These empty states used to be sticky when we had a long sidebar on the page, so that clearing the filters was easy. No longer needed.